core/templates/domain/collection/editable-collection-backend-api.service.ts

```diff
@@ -57,6 +58,7 @@ interface EditCollectionPropertyChange {
   'old_value': Object;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this required now?

Reply:
![Screenshot from 2021-02-26 00-56-42](https://user-images.githubusercontent.com/24855641/109206158-8519e580-77cd-11eb-8bec-5e7b191033a9.png)
The issue was that `this.undoRedoService.getCommittableChangeList()` function used to return a value of type `BackendChangeObject[]`. But the third parameter of `updateCollectionAsync` function excepts a value of type `CollectionChange[]`. Typescript was showing error that you cannot pass this value in the function.
So, I changed the parameter of `updateCollectionAsync` to type `BackendChangeObject[]`. Then `CollectionObject[]` interface become `un-used`. I want to ask the reviewers what to do in this situation.
Weather  to complelety remove the `CollectionChange[]` interface or to do something else.
So, to just pass the pre_commit checks I used `// eslint-disable-next-line @typescript-eslint/no-unused-vars` here.

Reply:
Removed CollectionChange interface.